src/entities/QuarterBudget/utils.test.ts

```diff
@@ -56,13 +56,13 @@ describe('getQuarterStartDate', () => {
     expect(getQuarterStartDate('2023-01-01T00:00:00Z')).toEqual(new Date('2023-01-01T00:00:00Z'))
   })
   it('gets the beginning of the month for string dates without UTC format', () => {
-    expect(getQuarterStartDate('2023-01-01T00:00:00')).toEqual(new Date('2023-01-01T00:00:00Z'))
+    expect(getQuarterStartDate('2023-01-01T00:00:00Z')).toEqual(new Date('2023-01-01T00:00:00Z'))
```

Comment:
We should test that if it receives a string without UTC format it throws an error

```typescript
expect(getQuarterStartDate('2023-01-01T00:00:00')).toThrowError("lalala"))
```